BuildForSDG/Team-037-Product-backend
# Ft user can edit profile

#### What does this PR do?
Feat: User Can Update Profile

#### How should this be manually tested?
Tell us how other people can test out this your feature to see if it's working as expected.
- git clone the repo
- npm i
- run: npm run dev and npm run test
![Capture56](https://user-images.githubusercontent.com/38535951/83328930-8342e600-a27e-11ea-834d-65a40742f3ce.PNG)
![Capture57](https://user-images.githubusercontent.com/38535951/83328937-88079a00-a27e-11ea-8d08-ce86fa1b6a4b.PNG)

#### Screenshots
take pictures of the implementation of the project and paste them here


Please clear out all the descriptions, including this one as soon as you're done.
